src/common/Routes.ts

```diff
@@ -1,9 +1,8 @@
 export enum RoutePaths {
-  DASHBOARD = "/dashboard",
-  HABIT_TRACKER = "/habit-tracker",
-  RECOMMENDATIONS = '/recommendations',
-  START_QUESTIONNAIRE = '/start-questionnaire',
-  QUESTIONNAIRE = '/questionnaire',
-  LOGIN = '/login',
-  LANDING = '/landing'
+  DASHBOARD = "/OralCheckr/dashboard",
+  HABIT_TRACKER = "/OralCheckr/habit-tracker",
+  RECOMMENDATIONS = '/OralCheckr/recommendations',
+  START_QUESTIONNAIRE = '/OralCheckr/start-questionnaire',
+  QUESTIONNAIRE = '/OralCheckr/questionnaire',
```

Comment:
You have two routes, one is `questionnaire` and another is `start-questionnaire`.
Let's keep only one route called: `questionnaire`.

And as you click Next on each question, the route will change. You will have to use "Route Parameters" for this. Check out Route Parameters, and we can talk about it in the next meeting.

So, route for question 1 will look like: `/questionnaire/1`
for question 2, it will look like: `/questionnaire/2` and so on.